ska_ftp/tests/test_basic.py

```diff
@@ -90,7 +89,7 @@ def test_delete_when_ftp_session_already_gone(capfd):
 def test_parse_netrc():
     cwd = os.path.dirname(__file__)
     netrc = ska_ftp.parse_netrc(os.path.join(cwd, 'netrc'))
-    assert netrc == {'test1': {'account': None,
+    assert netrc == {'test1': {'account': '',
```

Comment:
Not sure if this should be updated to have different values for different versions of netrc . I think what we're seeing lines up with https://docs.python.org/3/library/netrc.html "changed in version 3.10" .

Reply:
I haven't looked in detail, but my first reaction is that we should try to maintain API compatibility in `ska_ftp`, **if possible**. If it is a huge pain then we can reconsider.

Reply:
So would you want to code *around* the change in netrc.netrc to introduce api back compatibility for ska_ftp.parse_netrc?

Reply:
Thinking about this some more, we are unlikely to see this cause much of problem in the wild. Basically our users are going to either have a complete netrc entry or not all.

Reply:
Indeed, I couldn't think of a use case where we would be asking the netrc file for credentials when the user or password (credential pair) is incomplete.

Reply:
I would prefer to have this test compare to two possible reference results, one with `None` and the other with `''`. Either that, or do something to figure out which netrc version is present, and then use the right reference reult.

That way we can merge now instead of juggle that together with the speedy transition.